_posts/2020-04-22-#17994.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: pr
+date: 2020-04-22
+title: "Flush undo files after last block write"
+pr: 17994
+authors: [kallewoof]
+components: ["validation"]
+host: vasild
+status: upcoming
+commit:
+---
+
+## Notes
+
+In this Review Club we will dive into how blocks are downloaded and stored on disk. How disk access is arranged and a bug it has caused.
+
+Some concepts that would help to grasp what's going on:
+
+- [Initial Block Download](https://bitcoin.org/en/glossary/initial-block-download)
+
+- "block undo" - all the information that is necessary to revert a block, if a chain reorganization happens, for example.
+
+- [class FlatFileSeq](https://doxygen.bitcoincore.org/class_flat_file_seq.html)
```

Comment:
could you add a sentence of context about what is notable about this class?

Reply:
Done